src/Core/Models/Byte.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.
+ * Licensed under the MIT License.  See License in the project root
+ * for license information.
+ */
+
+namespace Microsoft\Graph\Core\Models;
+
+use InvalidArgumentException;
+use JsonSerializable;
+
+class Byte implements JsonSerializable
```

Comment:
(Applies to all classes & methods.)
Consider adding PHPDoc descriptions above classes & methods with a short description, params, return types, exceptions thrown etc. You can check out the other classes in the repo